versioned_docs/version-v0.11.0/int/faq/errors.mdx

```diff
@@ -0,0 +1,351 @@
+    </ul>
+  </details>
+  <details className="details">
+    <summary>
+      <h4>
+        <code>N/A</code> & <code>No data</code> in validator info panel
+      </h4>
+    </summary> Can be linked to the <a href="https://github.com/ObolNetwork/charon-distributed-validator-node#teku-keystore-file-error">Teku Keystore issue</a>.
+  </details>
+</details>
+<details open className="details">
+  <summary>
+    <h2 id="prometheus">Prometheus</h2>
+  </summary>
+  <details className="details">
+    <summary>
+      <h4>
+        <code>Unauthorized: authentication error: invalid token</code>
+      </h4>
+    </summary> Get the monitoring token and following <a href="https://github.com/ObolNetwork/charon-distributed-validator-node#step-6-leader-adds-central-monitoring-token">Step 6 of the quickstart</a>. This is not mandatory and should not prevent the cluster from running. Only the cluster captain/leader should do this step and resolve this error. Check <a href="https://discord.com/channels/849256203614945310/1003921582965674104/1005569855698768066">these instructions</a> from our Discord to get the monitoring token.
+  </details>
+</details>
+<details open className="details">
+  <summary>
+    <h2 id="docker"> Docker </h2>
+  </summary>
+  <details className="details">
+    <summary>
+      <h4> How to fix <code>permission denied</code> errors? </h4>
+    </summary> Permission denied errors can come up in a variety of manners, particularly on Linux and WSL for Windows systems. In the interest of security, the charon docker image runs as a non-root user, and this user often does not have the permissions to write in the directory you have checked out the code to. This can be generally be fixed with some of the following: <ul>
+      <li>Running docker commands with <code>sudo</code>, if you haven't <a href="https://docs.docker.com/engine/install/linux-postinstall/">setup docker to be run as a non-root user</a>. </li>
+      <li>Changing the permissions of the <code>.charon</code> folder with the commands: </li>
+      <ul>
+        <li>
+          <code>mkdir .charon</code> (if it doesn't already exist)
+        </li>
+        <li>
+          <code>sudo chmod -R 666 .charon</code>
+        </li>
+      </ul>
+    </ul>
+  </details>
+  <details className="details">
+    <summary>
+      <h4> I see a lot of errors after running <code>docker-compose up</code>
+      </h4>
+    </summary> It's because both geth and lighthouse start syncing and so there's connectivity issues among the containers. Simply let the containers run for a while. You won't observe frequent errors when geth finishes syncing. You can also add a second beacon node endpoint for something like infura by adding a comma separated API URL to the end of <code>CHARON_BEACON_NODE_ENDPOINTS</code> in the docker-compose(./docker-compose.yml#84).
+  </details>
+</details>
```

Comment:
<details className="details">
    <summary>
      <h4> How do I fix the <code>plugin "loki" not found</code> error?
      </h4>
    </summary> If you get the following error when calling `docker-compose up`:

`Error response from daemon: error looking up logging plugin loki: plugin "loki" not found`, then it probably means that the Loki docker driver isn't installed. In that case, run the following command to install loki:

```shell
docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
```
  </details>
</details>